sqlitecluster/SQLite.cpp

```diff
@@ -676,7 +676,7 @@ int SQLite::commit(const string& description) {
         int framesCheckpointed = 0;
         uint64_t start = STimeNow();
         int result = sqlite3_wal_checkpoint_v2(_db, 0, SQLITE_CHECKPOINT_PASSIVE, &walSizeFrames, &framesCheckpointed);
-        SINFO("[checkpoint] Checkpoint complete. Result: " << result << ". Total frames checkpointed: "
+        SDEBUG("[checkpoint] Checkpoint complete. Result: " << result << ". Total frames checkpointed: "
```

Comment:
nab, this has helped diagnose many problems with checkpointing in the past

Reply:
It's pretty redundant now with WAL2 where we do a lot less with checkpointing.